app/scripts/modules/core/whatsNew/whatsNew.read.service.spec.ts

```diff
@@ -0,0 +1,60 @@
+import {mock, IHttpBackendService} from 'angular';
+import {NetflixSettings} from 'netflix/netflix.settings';
+import {WHATS_NEW_READ_SERVICE, WhatsNewReader, IGistApiResponse, IWhatsNewContents} from './whatsNew.read.service';
+
+describe('Service: whatsNew reader ', () => {
+  let reader: WhatsNewReader;
+  let $http: IHttpBackendService;
+
+  beforeEach(
+    mock.module(
+      WHATS_NEW_READ_SERVICE
+    )
+  );
+
+  beforeEach(mock.inject((whatsNewReader: WhatsNewReader, $httpBackend: IHttpBackendService) => {
+    reader = whatsNewReader;
+    $http = $httpBackend;
+  }));
+
+  describe('getWhatsNewContents', () => {
+    let url: string;
+    beforeEach(() => {
+      const gistId = NetflixSettings.whatsNew.gistId;
+      url = ['https://api.github.com/gists/', gistId].join('');
+    });
+
+    it ('returns file contents with lastUpdated', () => {
+
+      let result: IWhatsNewContents = null,
+        response: IGistApiResponse = {
+          'updated_at': '1999',
+          files: {},
+        };
+
+      response.files[NetflixSettings.whatsNew.fileName] = {
```

Comment:
Just a head's up that settings appear to be stateful between tests in the same test file (tests look to be executed top-down).  I tested across different files and while it looks like it's ok, I don't think it's safe to rely on the test iteration order being deterministic.